cluster_loader/cluster_loader.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+import os
+import argparse
+import configparser
+import logging
+
+import trigger_cluster_loader
+
+logger = logging.getLogger("snafu")
+
+class cluster_loader_wrapper():
```

Comment:
Kind of a nit but to keep in line with the other wrappers should the file be cluster_loader_wrapper.py instead of cluster_loader.py?

Reply:
Nice catch, i'll be going through and fixing all the other wrappers but this is how i think the file naming should be, wrapper is a class not a file if it makes sense.